forge-gui/res/cardsfolder/upcoming/rendmaw_creaking_nest.txt

```diff
@@ -0,0 +1,12 @@
+Name:Rendmaw, Creaking Nest
+ManaCost:3 B G
+Types:Legendary Artifact Creature Scarecrow
+PT:5/5
+K:Menace
+K:Reach
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ DBTokens | TriggerDescription$ When CARDNAME enters each player creates a tapped 2/2 black Bird creature token with flying. The tokens are goaded for the rest of the game.
```

Comment:
when a trigger is caused by 2+ different events we still use the printed description
but we add Secondary$ param after the first one so it doesn't show up ingame

Reply:
both need the same TriggerDescription
(relevant if e.g. you're asked to order simultaneous stack entry)